bundle/language_de.properties

```diff
@@ -489,6 +489,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = positiv
 angal.lab.printreport                                                            = Bericht drucken
 angal.lab.printtable                                                             = Tabelle drucken
+angal.lab.print																	 = Drucken
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d